src/app/shared/components/status-info-card/status-info-card.component.html

```diff
@@ -1,12 +1,13 @@
 <div *ngIf="application; else cardFormat;">
-  <div class="application-popup-status {{application.status}}">
-    <span> <i class={{applicationIcons[application.status]}}></i></span>
-    <span> {{applicationTitles[application.status]}}</span>
-  </div>
-  <div class="application-info-menu application-info-description-card">({{
-    applicationStatusDescription[application.status] }})</div>
-    <p *ngIf="application.rejectionMessage" class="application-info-menu">{{application.rejectionMessage}}</p>
+    <div class="application-popup-status {{application.status}}">
+      <span> <i class={{applicationIcons[application.status]}}></i></span>
+      <span> {{applicationTitles[application.status]}}</span>
+    </div>
+    <div class="application-info-menu application-info-description-card">({{
+      applicationStatusDescription[application.status] }})</div>
+      <p *ngIf="application.rejectionMessage" class="application-info-menu">{{application.rejectionMessage}}</p>
```

Comment:
in ts file     this.status = this.application.isBlocked ? ApplicationStatus.Blocked : ApplicationStatus[this.application.status];